source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -118,41 +119,30 @@ generated by oVirt:
 
 ![](../../../../images/wiki/upload-ova.png)
 
-1. The front end retrieves the OVF configuration from the OVA.
-2. The front end sends the OVF configuration to the back end.
-3. The back end parses the OVF configuration and returns a VM to the
-   front end.
-4. The front end sends (optionally customized) parameters of the import
-   process to the back end.
-5. The back end persists the VM into the database.
-6. The back end triggers the creation of the target VM disks on the
-   storage domain.
-7. The back end sends the import parameters - including the destination
-   disks - to imageio-proxy, which sends them to imageio.
-8. Imageio-proxy sends the engine a URL to write to. This URL is sent to
-   the front end.
-9. The front end streams the **whole OVA** to imageio-proxy, which
-   streams it to imageio.
-10. Imageio reads the data and according to the import parameters
-    streams the data into the corresponding target disks.
+1. The frontend retrieves the OVF configuration from the OVA.
```

Comment:
Lets remove these changes (frontend/backend), they are not related to the flow change.